src/widgets/dropdown/template.tpl

```diff
@@ -1,14 +1,44 @@
 <ul class="CDB-Dropdown-list CDB-Text CDB-Size-medium">
   <li class="CDB-Dropdown-item">
-    <button class="CDB-Dropdown-link js-toggle">Toggle widget</button>
+    <button class="CDB-Dropdown-link js-toggleCollapsed">
+      Toggle widget
+        <div>
+        <% if (collapsed) { %>
```

Comment:
Yes, sorry, you are right… I'm not referring to semistandard, when I say "linter" I mean that the code doesn't compile due to an internal validation.